### PR TITLE
Update deployment-integrations.adoc

### PR DESCRIPTION
**Pull request by thomasr8**
# Description
replaced references to legacy images with next-gen images

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/deployment-integrations.adoc [hunk 1]
```diff
@@ -88,11 +88,11 @@ workflows:
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4-node
+      - image: cimg/ruby:2.4-node
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine
+      - image: cimg/postgres:9.4.12-alpine
```
Comment (rosieyohannan) [inline jekyll/_cci2/deployment-integrations.adoc:95]: This image doesn't exist - https://circleci.com/developer/images/image/cimg/postgres

File: jekyll/_cci2/deployment-integrations.adoc [hunk 2]
```diff
@@ -120,11 +120,11 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4-node # primary container - where job steps are run
+      - image: cimg/ruby:2.4-node # primary container - where job steps are run
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine # services container
+      - image: cimg/postgres:9.4.12-alpine # services container
```
Comment (felicianotech) [inline jekyll/_cci2/deployment-integrations.adoc:127]: -alpine needs to go.

File: jekyll/_cci2/deployment-integrations.adoc (hunk 1 above)
Comment (felicianotech) [reply jekyll/_cci2/deployment-integrations.adoc:95]: `9.6` is a tag

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/deployment-integrations.adoc (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/deployment-integrations.adoc]: @felicianotech I went ahead and replaced the this with 9.6. Is that correct?

File: jekyll/_cci2/deployment-integrations.adoc (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/deployment-integrations.adoc]: @felicianotech I went ahead and replaced 9.4.12-alpine with 9.6. Is this what you were referring to as the change that was needed?

**Review by felicianotech — APPROVED** [review]

**Pull request merged**